apps/src/code-studio/components/header/ScriptName.jsx

```diff
@@ -32,23 +75,60 @@ class ScriptName extends React.Component {
         href={this.props.href}
         className={className}
         style={
-          this.props.showProjectUpdatedAt ? styles.scriptLinkWithUpdatedAt : {}
+          this.props.showProjectUpdatedAt
+            ? {...styles.scriptLinkWithUpdatedAt}
+            : {}
         }
+        title={this.props.name}
       >
         {this.props.name}
       </a>
     );
   }
 
   render() {
+    const fullWidth = this.getFullWidth();
+    const actualWidth = this.props.width;
+
+    const vignetteStyle =
+      actualWidth < fullWidth
+        ? this.props.isRtl
+          ? headerVignetteStyles.left
+          : headerVignetteStyles.right
+        : null;
+
+    console.log('ScriptName render', this.props.width);
+
     if (!this.props.showProjectUpdatedAt) {
-      return this.renderScriptLink();
+      return (
+        <div style={{...styles.headerContainer, height: 18}}>
```

Comment:
There are a few places where height is set to 18 - would it be worth setting something like a `headerHeight` variable somewhere more generic and using in each of these places?

Reply:
That's a great point.  It looks like this change is pretty stable so I might punt to a followup.